### PR TITLE
Rewording doc string for clip_grad_norm_

### PR DESCRIPTION
**Pull request by akshay-raj-dhamija**
The doc string for `torch.nn.utils.clip_grad_norm_` needed some clarity, it was earlier unclear that the norm was being computed over the norms of individual gradient parameters.

### CODE REVIEW

**Review by mikaylagawarecki — APPROVED** [review]
Is this clarification due to precision differences between the norm(cat(grads)) (which the docstring seems to suggest now) and norm(cat(map(norm, grads)) (the actual implementation)?

**Comment by akshay-raj-dhamija** [comment]
Yes the implementation v/s the doc string approach could have different precision. Even though theoretically they are the same.
